src/Files.App/Utils/Shell/LaunchHelper.cs

```diff
@@ -34,7 +34,11 @@ public static Task<bool> LaunchAppAsync(string application, string arguments, st
 		public static Task<bool> RunCompatibilityTroubleshooterAsync(string filePath)
 		{
 			var compatibilityTroubleshooterAnswerFile = Path.Combine(Path.GetTempPath(), "CompatibilityTroubleshooterAnswerFile.xml");
-			File.WriteAllText(compatibilityTroubleshooterAnswerFile, string.Format("<?xml version=\"1.0\" encoding=\"UTF-8\"?><Answers Version=\"1.0\"><Interaction ID=\"IT_LaunchMethod\"><Value>CompatTab</Value></Interaction><Interaction ID=\"IT_BrowseForFile\"><Value>{0}</Value></Interaction></Answers>", filePath));
+
+			SafetyExtensions.IgnoreExceptions(() =>
+			{
+				File.WriteAllText(compatibilityTroubleshooterAnswerFile, string.Format("<?xml version=\"1.0\" encoding=\"UTF-8\"?><Answers Version=\"1.0\"><Interaction ID=\"IT_LaunchMethod\"><Value>CompatTab</Value></Interaction><Interaction ID=\"IT_BrowseForFile\"><Value>{0}</Value></Interaction></Answers>", filePath));
+			});
```

Comment:
Ignoring the exception here would not make the next process work.
I think we should continue the process by trying another file name instead of ignoring the exception.

Reply:
Is this a file that we're generating from Files?

Reply:
I believe so and any file name is acceptable.

Reply:
I added a try/catch to try once more with a different file name.